CodingChallenges/CC_048_TweetsByMonth/Processing/CC_048_TweetsByMonth/CC_048_TweetsByMonth.pde

```diff
@@ -0,0 +1,137 @@
+// Obamathon
+// https://github.com/ITPNYU/Obamathon
+// YouTube video tutorial: https://youtu.be/UrznYJltZrU
+// Javascript transcription: Chuck England
+
+import java.util.*;
+import java.text.*;
+
+// Class to hold the monthly totals and word counts.
+class Count { 
+  int total;
+  Map<String, Integer> words;
```

Comment:
Would it be possible to use Processing's native `IntDict` class? 
https://processing.org/reference/IntDict.html

Reply:
I think you pointed one of these out before. I will switch this one out and try to remember it for the future. :)